packages/flutter/lib/src/widgets/ticker_provider.dart

```diff
@@ -92,24 +110,114 @@ class TickerMode extends StatefulWidget {
+        .dependOnInheritedWidgetOfExactType<_EffectiveTickerMode>();
+    return widget?.values ?? TickerModeData.fallback;
+  }
+
+  /// Obtains a [ValueListenable] from the [TickerMode] surrounding the `context`,
+  /// which indicates whether tickers are enabled in the given subtree.
+  ///
+  /// When that [TickerMode] enabled or disabled tickers, the listenable notifies
+  /// its listeners.
+  ///
+  /// While the [ValueListenable] is stable for the lifetime of the surrounding
+  /// [TickerMode], calling this method does not establish a dependency between
+  /// the `context` and the [TickerMode] and the widget owning the `context`
+  /// does not rebuild when the ticker mode data changes. This is preferable
+  /// when the ticker mode does not impact what is currently rendered on screen,
+  /// e.g. because it is only used to mute/unmute a [Ticker]. Since no dependency
+  /// is established, the widget owning the `context` is also not informed when
+  /// it is moved to a new location in the tree where it may have a different
+  /// [TickerMode] ancestor. When this happens, the widget must manually
+  /// unsubscribe from the old listenable, obtain a new one from the new ancestor
+  /// [TickerMode] by calling this method again, and re-subscribe to it
+  /// [StatefulWidget]s can, for example, do this in [State.activate],
+  /// which is called after the widget has been moved to a new location.
+  ///
+  /// Alternatively, [of] can be used instead of this method to create a
+  /// dependency between the provided `context` and the ancestor [TickerMode].
+  /// In this case, the widget automatically rebuilds when the ticker mode
+  /// changes or when it is moved to a new [TickerMode] ancestor, which
+  /// simplifies the management cost in the widget at the expensive of some
+  /// potential unnecessary rebuilds.
+  ///
+  /// In the absence of a [TickerMode] widget, this function returns a
+  /// [ValueListenable], whose [ValueListenable.value] is
+  /// [TickerModeData.fallback].
+  static ValueListenable<TickerModeData> getValuesNotifier(BuildContext context) {
+    final _EffectiveTickerMode? widget = context
+        .getInheritedWidgetOfExactType<_EffectiveTickerMode>();
+    return widget?.valuesNotifier ??
+        const _ConstantTickerModeDataListenable(TickerModeData.fallback);
+  }
+
+  /// Creates a [TickerMode] that overrides the ambient ticker mode values.
+  ///
+  /// The given `enabled` and `forceFrames` override the ambient values when not null;
+  /// otherwise the ambient values are preserved.
+  static Widget merge({Key? key, bool? enabled, bool? forceFrames, required Widget child}) {
+    return Builder(
+      builder: (BuildContext context) {
+        final _EffectiveTickerMode? parent = context
+            .dependOnInheritedWidgetOfExactType<_EffectiveTickerMode>();
+        final bool parentEnabled = parent?.enabled ?? true;
+        final bool parentForce = parent?.forceFrames ?? false;
```

Comment:
Consider using `TickerModeData.fallback` here as well:

```suggestion
        final bool parentEnabled = parent?.enabled ?? TickerModeData.fallback.enabled;
        final bool parentForce = parent?.forceFrames ?? TickerModeData.fallback.forceFrames;
```

This would ensure `merge`'s logic doesn't get stale if we change the default values for whatever reason.

(The Dart formatting of this suggestion is likely wrong)